distsql/request_builder_test.go

```diff
@@ -252,17 +252,17 @@ func TestRequestBuilder1(t *testing.T) {
 				EndKey:   kv.Key{0x74, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc, 0x5f, 0x72, 0x80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x23},
 			},
 		},
-		Cacheable:      true,
-		KeepOrder:      false,
-		Desc:           false,
-		Concurrency:    variable.DefDistSQLScanConcurrency,
-		IsolationLevel: 0,
-		Priority:       0,
-		NotFillCache:   false,
-		SyncLog:        false,
-		Streaming:      false,
-		ReplicaRead:    kv.ReplicaReadLeader,
-		TxnScope:       oracle.GlobalTxnScope,
+		Cacheable:        true,
+		KeepOrder:        false,
+		Desc:             false,
+		Concurrency:      variable.DefDistSQLScanConcurrency,
+		IsolationLevel:   0,
+		Priority:         0,
+		NotFillCache:     false,
+		SyncLog:          false,
+		Streaming:        false,
+		ReplicaRead:      kv.ReplicaReadLeader,
+		ReadReplicaScope: oracle.GlobalTxnScope,
```

Comment:
IMO，`ReplicaScope` and `TxnScope` sound confusing, maybe `ReadReplicaScope: Globally/Locally` more suitable. or add some comments.

Reply:
updated.